weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/traceServerClientTypes.ts

```diff
@@ -420,3 +420,11 @@ export type TableCreateRes = {
   digest: string;
   row_digests: string[];
 };
+
+export type ProjectCheckReq = {
+  project_id: string;
+};
+
+export type ProjectCheckRes = {
+  has_data: boolean;
```

Comment:
I'm guessing that the purpose of this endpoint is that checking if data exists is faster than actually pulling the project data and sending it over the wire?